tilleuls/url-signer-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
```

Comment:
You should add the env var to the manifest too. So Flex will automatically add it to `.env`.